backend/src/apiserver/server/recurring_run_server.go

```diff
@@ -0,0 +1,344 @@
+// Copyright 2022 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	apiv2beta1 "github.com/kubeflow/pipelines/backend/api/v2beta1/go_client"
+	"github.com/kubeflow/pipelines/backend/src/apiserver/common"
+	"github.com/kubeflow/pipelines/backend/src/apiserver/model"
+	"github.com/kubeflow/pipelines/backend/src/apiserver/resource"
+	"github.com/kubeflow/pipelines/backend/src/common/util"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/robfig/cron"
+	authorizationv1 "k8s.io/api/authorization/v1"
+)
+
+// Metric variables. Please prefix the metric names with recurring_run_server_.
+var (
+	// Used to calculate the request rate.
+	createRecurringRunRequests = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "recurring_run_server_create_requests",
```

Comment:
Creating new counter will make us lose track of the new APIs until we fix our monitoring side. How did we handle this with experiment service? If we were reusing the same counter, maybe keep consistency for now and separate v1 and v2 in a batch effort.

Reply:
With experiment, we indeed used the same counters. I will change the code so we use the same counter for both v1 and v2 apis for now.

Reply:
Sorry, I commented the part when removing the recurring run counter. I will remove the whole thing.